build/targets/DefaultItems.targets

```diff
@@ -0,0 +1,25 @@
+<Project>
+  <PropertyGroup>
+    <SharedSourcesFolder>$(RepositoryRoot)shared/</SharedSourcesFolder>
+  </PropertyGroup>
+
+  <!-- DisableDefaultItems only when you really want to control what is built and tested. Otherwise, use the Exclude* item groups -->
+  <ItemGroup Condition="'$(DisableDefaultItems)' != 'true'">
+    <Solutions Include="$(RepositoryRoot)\*.sln" />
+    <Solutions Remove="@(ExcludeSolutions)" />
+
+    <ProjectsToPack Include="$(RepositoryRoot)src\*\*.csproj" />
+    <ProjectsToPack Remove="@(ExcludeFromPack)" />
+
+    <SharedSourceDirectories Include="$([System.IO.Directory]::GetDirectories(&quot;$(SharedSourcesFolder)&quot;, '*.Sources'))" Condition="Exists('$(SharedSourcesFolder)')" />
+    <SharedSourceDirectories Remove="@(ExcludeSharedSourceDirectories)" />
+
+    <!-- put unit test projects ahead of functional tests -->
+    <ProjectsToTest Include="$(RepositoryRoot)test\*\*.Tests.csproj" />
```

Comment:
👍 

Although, the thinking behind not doing `Condition="'@(ProjectsToTest)'==''"` is that the most common scenario here is that we will want to exclude a project from testing `ExcludeFromTest` that was picked up by default globs. Redefining the default globs is cumbersome.